<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Crypto</class>
 <widget class="QMainWindow" name="Crypto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Цифровая подпись</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>761</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Подпись</string>
    </property>
    <widget class="QTextEdit" name="textEdit_FileNameSign">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Sign">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Подписать</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_FileNameKeys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fileInputForSign">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/explorer.png</normaloff>../../Downloads/explorer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fileInputForSign_Keys">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>120</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/explorer.png</normaloff>../../Downloads/explorer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_FileNameParams">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fileInputForSign_Params">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/explorer.png</normaloff>../../Downloads/explorer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Ключи</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Параметры</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>761</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Проверка подписи</string>
    </property>
    <widget class="QPushButton" name="pushButton_fileInputForCheck">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/explorer.png</normaloff>../../Downloads/explorer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CheckSign">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>130</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Проверить подпись</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_FileNameCheckSign">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_FileNameCheckSign_Keys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_FileNameCheck_Params">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fileInputForCheck_Keys">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>130</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/explorer.png</normaloff>../../Downloads/explorer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_fileInputForCheck_Params">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/explorer.png</normaloff>../../Downloads/explorer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ключи</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Параметры</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Генерация ключей</string>
    </property>
    <widget class="QPushButton" name="pushButton_fileInputForKeys">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../Downloads/explorer.png</normaloff>../../Downloads/explorer.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Keys">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Генерация ключей</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_keys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>500</width>
       <height>50</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
